ui/src/chatbot.tsx

```diff
@@ -134,7 +143,22 @@ export const XChatbot = (props: Chatbot) => {
         setIsInfiniteLoading(true)
       }
     }, [props.events, props.name]),
-    onDataChange = React.useCallback(() => { if (props.data) setMsgs(processData(props.data)) }, [props.data])
+    onDataChange = React.useCallback(() => { if (props.data) setMsgs(processData(props.data)) }, [props.data]),
+    handleFeedback = (positive: B, id: I) => {
+      if (props.events?.includes('feedback')) {
```

Comment:
Isn't this already taken care of within the render logic? The buttons shouldn't even render if this is not true, right?